.mvn/jvm.config

```diff
@@ -11,3 +11,4 @@
 --add-opens=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED
 -XX:+UnlockDiagnosticVMOptions
 -XX:+ExitOnOutOfMemoryError
+-XX:G1NumCollectionsKeepPinned=10000000
```

Comment:
would that be wise to fail on startup if we see that JVM version is one without fix and we do not have `-XX:G1NumCollectionsKeepPinned=10000000` set?